### PR TITLE
Display remote folder size

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Folder size is now shown by remote computed folder size.

originally opened by @tobiasKaminsky on oC repo, rebased to latest master now.

According to the original discussion in the PR 1655 this will not work for Servers with oC version below 8 so it will always work for Nextcloud version > 9.x. So we are imho save here.

Please review @przybylski @tobiasKaminsky (I will do one too).

Since this is a minot change we might add this to the 1.2.0 release. Comments? :)

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Code LGTM

File: src/com/owncloud/android/utils/FileStorageUtils.java [hunk 1]
```diff
@@ -278,39 +282,38 @@ else if (o1.isFolder()) {
...
-//            }
-//        });
-//        
-//        return files;
-//    }
+    /**
+     * Sorts list by Size
+     */
+    public static Vector<OCFile> sortBySize(Vector<OCFile> files){
+        final Integer val;
```
Comment (przybylski) [inline src/com/owncloud/android/utils/FileStorageUtils.java]:
```java
final int multiplier = mSortAscending ? 1 : -1;
```

File: src/com/owncloud/android/utils/MimeType.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package com.owncloud.android.utils;
+
+/**
+ * Class containing the mime types.
+ */
+public class MimeType {
+    public static final String DIRECTORY = "DIR";
```
Comment (przybylski) [inline src/com/owncloud/android/utils/MimeType.java]: 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍

**Comment by przybylski** [comment]
👍

File: src/com/owncloud/android/utils/MimeType.java (hunk 2 above)
Comment (AndyScherzinger) [inline src/com/owncloud/android/utils/MimeType.java]: Hahahahaha :) I thought you might react like this to this change ;)

File: src/com/owncloud/android/utils/FileStorageUtils.java (hunk 1 above)
Comment (AndyScherzinger) [inline src/com/owncloud/android/utils/FileStorageUtils.java]: fixed by https://github.com/nextcloud/android/pull/123/commits/174fa1d7b7b6edbca7725ee8928b272c7275a7f9

**Comment by AndyScherzinger** [comment]
Fixed @przybylski code review comments :+1:

**Comment by przybylski** [comment]
👍

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky please check and merge at will ;) Originally it is your PR, so it is yours to push the button :)

**Comment by tobiasKaminsky** [comment]
LGTM
this thing drives me crazy...

**Comment by tobiasKaminsky** [comment]
Is this working for you? For me, I do not see any remote folder size :/
@AndyScherzinger @przybylski

**Comment by AndyScherzinger** [comment]
Erm...just checked and I don't see any either :(

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky I fixed this here: #235 - please review and merge :)

**Pull request merged**